metadata/ranges.json

```diff
@@ -3649,4 +3848,4 @@
 		"range_end": "U+E43F",
 		"range_start": "U+E420"
 	}
-}
+}
```

Comment:
ditto

Reply:
Similarly, I'm not sure what you're questioning here. Sorry for being slow!

Reply:
see abobe

Reply:
All three of these JSON files finish with a newline. Is that a problem?

Reply:
Well, that diff here on GitHub indicates the opposite

Reply:
All of these JSON files are UTF-8 and use LF for line endings. In my editor (VS Code) they all show that the end with a newline.

Reply:
It most probably doesn't matter at all, but there clearly is a difference before and after the change. Might well have been introduced by #135.
And indeed has...

Reply:
@mirabilos Could you please take a look at the JSON files in this PR and check that you're happy with the whitespace approach, taking into consideration the changes you made in #135?

Reply:
As far as I can tell the whitespaces are still tabs, which was the core of @mirabilos' changes